packages/backend/src/peripherals/discord/DiscordClient.ts

```diff
@@ -13,18 +13,20 @@ export class DiscordClient {
   constructor(
     private readonly httpClient: HttpClient,
     private readonly discordToken: string,
-    private readonly publicChannelId: string,
+    private readonly publicChannelId: string | undefined,
     private readonly internalChannelId: string,
   ) {}
 
   async sendMessage(message: string, channel: 'PUBLIC' | 'INTERNAL') {
     if (message.length > MAX_MESSAGE_LENGTH) {
       throw new Error(`Discord error: Message size exceeded (2000 characters)`)
     }
-    const channelId =
-      channel === 'PUBLIC' ? this.publicChannelId : this.internalChannelId
-
-    return this.send(message, channelId)
+    if (channel === 'PUBLIC' && this.publicChannelId) {
+      return this.send(message, this.publicChannelId)
+    }
```

Comment:
Maybe log about the skip of the public channel? We don't want to forget adding publicChannelId in prod

Reply:
In production the `config.production.ts` enforces having `process.env.PUBLIC_DISCORD_CHANNEL_ID`